common/app/views/fragments/newHeader.scala.html

```diff
@@ -15,8 +15,8 @@ <h1 class="u-h">The Guardian</h1>
             <span>UK Edition</span>
         </button>
         <nav class="new-header__nav">
-            <a href="#main-menu" class="new-header__nav__menu-button js-change-link" tabindex="0">
-                <div class="new-header__burger-icon js-animate-menu">
+            <a href="#main-menu" class="new-header__nav__menu-button js-animate-menu" tabindex="0">
```

Comment:
Does this do anything without JS? If it purely gets intercepted, it should most likely be a `button` instead of an anchor.